notification-service/src/main/java/com/microcommerce/notificationservice/service/AccountCreatedEventHandlerService.java

```diff
@@ -0,0 +1,50 @@
+package com.microcommerce.notificationservice.service;
+
+import com.microcommerce.notificationservice.data.event.AccountCreatedEvent;
+import jakarta.mail.MessagingException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.context.Context;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class AccountCreatedEventHandlerService {
+    private static final String ACCOUNT_CONFIRMATION_EMAIL_TEMPLATE = "account_confirmation_template";
+    private static final String UTF_8_ENCODING = "UTF-8";
+    private static final String ACCOUNT_CONFIRMATION_URL = "http://localhost:8080/auth/confirm?token=";
```

Comment:
Values that change depending on the environment (in this case: host, port) should be moved to be in the .env file.

Reply:
Done.